examples/nspawn2.yaml

```diff
@@ -0,0 +1,7 @@
+---
+graph: mygraph
+resources:
+  nspawn:
+  - name: mgmt-nspawn1
```

Comment:
Not necessarily. I used nspawn1.yaml to start it, then nspawn2.yaml to stop it. If you want it to be 2, though, it doesn't matter to me.